dev/Vagrantfile

```diff
@@ -58,7 +58,6 @@ def define_chef_server(config, attributes)
              "opscode_erchef['db_pool_size']" => 10,
              "oc_id['db_pool_size']" => 10,
              "oc_bifrost['db_pool_size']" => 10 }
-      # TODO merge with other options nicely.
       json = simple_deep_merge(json, { "provisioning" => { "chef-server-config" => pg } })
```

Comment:
Did you mean to remove this line? I just see the TODO being removed without an actual change...